python/tk_framework_desktopserver/shotgun/scripts/execute_command.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) 2017 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import sys
+import cPickle
+import os
+
+def execute(config, project, name, entities, base_configuration, engine_name):
+    import sgtk
+
+    ms_flag = sgtk.platform.constants.LEGACY_MULTI_SELECT_ACTION_FLAG
+
+    # Setup the bootstrap manager.
+    toolkit_mgr = sgtk.bootstrap.ToolkitManager()
+    toolkit_mgr.allow_config_overrides = False
```

Comment:
it would be great if we could go through this use case again and why it's not working without this param. We should add comments to explain!